.github/workflows/reusable-release.yml

```diff
@@ -85,7 +85,7 @@ jobs:
         env:
           GITHUB_NPM_TOKEN: ${{ secrets.GITHUB_NPM_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - run: npm run build:docs
+      - run: node --run build:docs
```

Comment:
will this work if we pin the pipeline to an older version of Node?

Reply:
No, but do we care? Node 22 is the new LTS, so in my opinion we should just move everything to that anyway